test/unquote-properties-test.js

```diff
@@ -0,0 +1,13 @@
+var x = {
+  'quotedProp': 1,
+  unquotedProp: 2,
+  'quoted-prop': 3,
+  method() { return 4; },
```

Comment:
oh. Probably good to add a test for other computed properties as well, actually.

```
const dontDoThis = {
  [Math.random()]() { return 'oh no'; }
}
```